torch/_torch_docs.py

```diff
@@ -3286,6 +3286,36 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Regularized lower incomplete gamma function
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input})} \int_0^\text{other} t^{\text{input}-1} e^{-t} dt
```

Comment:
The input and output values in the expression also need subscripts.

Reply:
After this mathematical expression is a good opportunity to describe the constraints on the input, and what will happen if those constraints are violated.

Reply:
This needs to define the Gamma function. It's interesting that SciPy's documentation doesn't seem to define it, either.

Reply:
Is there any docs example in pytorch to explain the constraints?